packages/mui-material/src/TablePagination/TablePagination.d.ts

```diff
@@ -18,7 +18,10 @@ export interface LabelDisplayedRowsArgs {
 /**
  * This type is kept for compatibility. Use `TablePaginationOwnProps` instead.
  */
-export type TablePaginationBaseProps = Omit<TableCellProps, 'classes' | 'component' | 'children'>;
+export type TablePaginationBaseProps = Omit<
+  TableCellProps,
+  'classes' | 'component' | 'children' | 'variant'
+>;
```

Comment:
Roll this back. We don't need to change this.

Reply:
Thanks, I forgot that change. Rolled it back now.